docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,601 @@
+    - The "type" property is set to "button" which means that the control will be a ribbon button. You can also configure a task pane command to be executed from a menu item. See [Menu and menu items](#menu-and-menu-items).
+    - The "supertip.title" (maximum length: 64 characters) and "supertip.description" (maximum length: 128 characters) appear when the cursor is hovering over the button or menu.
+    - The "actionId" must be an exact match for the "runtimes.actions.id" that you set in [Configure the runtime for the task pane command](#configure-the-runtime-for-the-task-pane-command).
+
+    ```json
+    {
+        "id": "msgReadOpenPaneButton",
+        "type": "button",
+        "label": "Show Task Pane",
+        "icons": [
+            {
+                "size": 16,
+                "file": "https://localhost:3000/assets/icon-16.png"
+            },
+            {
+                "size": 32,
+                "file": "https://localhost:3000/assets/icon-32.png"
+            },
+            {
+                "size": 80,
+                "file": "https://localhost:3000/assets/icon-80.png"
+            }
+        ],
+        "supertip": {
+            "title": "Show Contoso Task Pane",
+            "description": "Opens the Contoso task pane."
+        },
+        "actionId": "ShowTaskPane"
+    }
+    ```
+
+You have now completed adding a task pane command to your add-in.  [Sideload and test it](../testing/test-debug-office-add-ins.md#sideload-an-office-add-in-for-testing).
+
+## Add a function command
+
+The following subsections explain how to include a [function command](../design/add-in-commands.md#types-of-add-in-commands) in an add-in.
+
+### Create the code for the function command
+
+1. Ensure that your source code includes a JavaScript or Typescript file with the function that you want to execute with your function command. The following is an example. Since this article is about creating add-in commands, and not about teaching the Office JavaScript Library, we provide it with minimal comments, but do note the following:
+
+    - For purposes of this article, the file is named commands.js.
+    - The function will cause a small notification to appear on an open email message with the text "Action performed".
+    - Like all code files that call APIs in the Office JavaScript Library, it must begin by [initializing the library](initialize-add-in.md). It does this by calling `Office.onReady`. 
+    - The last thing the file calls is [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member) to tell Office which function in the file should be executed when the UI for your function command is invoked. The function maps the function name to an action id that you configure in the manifest in a later step. If you define multiple function commands in the same file, your code must call `associate` for each one.
```

Comment:
Feel free to change this and the previous one back to "file", but it seems more correct to say "code". 
```suggestion
    - The last thing the code calls is [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member) to tell Office which function in the file should be executed when the UI for your function command is invoked. The function maps the function name to an action ID that you configure in the manifest in a later step. If you define multiple function commands in the same file, your code must call `associate` for each one.
```